addons/point_of_sale/models/pos_payment.py

```diff
@@ -18,7 +18,7 @@ class PosPayment(models.Model):
 
     name = fields.Char(string='Label', readonly=True)
     pos_order_id = fields.Many2one('pos.order', string='Order', required=True, index=True)
-    amount = fields.Monetary(string='Amount', required=True, currency_field='currency_id', readonly=True, help="Total amount of the payment.")
+    amount = fields.Monetary(string='Amount', required=True, currency_field='currency_id', help="Total amount of the payment.")
```

Comment:
Is it possible to make this readonly when the linked `pos_order_id` can't be modified?

Reply:
Its done in the view and I made a constrains